docs/content/installation/configuration.md

```diff
@@ -158,7 +152,32 @@ other storage classes in production deployments.
 | `REPLICA_STORAGE` | hostpathstorage | **Required** | Set to configure which storage definition to use when creating volumes used by PostgreSQL replicas on all newly created clusters. |
 | `WAL_STORAGE` |  |  | Set to configure which storage definition to use when creating volumes used for PostgreSQL Write-Ahead Log |
```

Comment:
I assume all vars defined in here will now be updated to their lowercase equivalent?

Reply:
Yeah the will need to be eventually. I may hold off until I get the helm stuff up